lib/ViewModels/ChartPanelViewModel.js

```diff
@@ -12,6 +12,9 @@ var combineData = require('../Core/combineData');
 var loadView = require('../Core/loadView');
 var removeView = require('../Core/removeView');
 var ChartViewModel = require('./ChartViewModel');
+var cContainer = document.getElementById('cesiumContainer');
```

Comment:
`Terria` knows the `currentViewer`, which surely has a reference to its DOM element somehow, but it's not really exposed in a useful way.  We could probably expose it.